test/passing/object.ml

```diff
@@ -6,6 +6,8 @@ let _ =
     (* some comment *)
     method! x = 2 [@@attr]
 
+    method x = (1 [@attr])
```

Comment:
These parentheses are not necessary.

Reply:
Is this fixed by https://github.com/ocaml-ppx/ocamlformat/pull/1291 ?

Reply:
It does not, only those on the LHS of sequences will be, i.e.

```ocaml
method a = (0 [@attr]) ; 1

let _ = (0 [@attr]) ; 1
```